accumulator/pollard.go

```diff
@@ -225,6 +225,7 @@ func (p *Pollard) rem2(dels []uint64) error {
 		}
 		hashDirt = nextHashDirt
 		nextHashDirt = []uint64{}
+		// fmt.Printf("done with row %d %s\n", h, p.toString())
```

Comment:
I think this was removed in a previous commit. Could you remove this from your branch